src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -135,6 +135,17 @@ export class CalciteRadioButton {
 
   private rootNode: HTMLElement;
 
+  //--------------------------------------------------------------------------
+  //
+  //  Public Methods
+  //
+  //--------------------------------------------------------------------------
+
+  @Method()
+  async setFocus(): Promise<void> {
+    focusElement(this.inputEl);
+  }
```

Comment:
Should we update the two other places in here that call `this.input.focus()` to use `focusElement()`?

Reply:
Nevermind, we're gonna ditch the `focused` attribute altogether in favor of `setFocus`

Reply:
I'll leave that to #1566 since it's tied to the `focused` prop.